Guesser/DoctrineFieldGuesser.php

```diff
@@ -62,7 +62,8 @@ public function getAllFields($class)
     public function getManyToMany($model, $fieldPath)
     {
         $resolved = $this->resolveRelatedField($model, $fieldPath);
-        return !$this->getMetadatas($resolved['class'])->isAssociationWithSingleJoinColumn($resolved['field']);
+        $metadata = $this->getMetadatas($resolved['class']);
+        return $metadata->hasAssociation($resolved['field']) && !$metadata->isAssociationWithSingleJoinColumn($resolved['field']);
```

Comment:
This seams like a BC-breaking behavior change. The only occurrence of usage seems to be [here](https://github.com/symfony2admingenerator/GeneratorBundle/blob/master/Resources/templates/CommonAdmin/ListAction/filters.php.twig#L88). Can you tell me if you tested this with the filters? I'm currently have no possibility to test...